### ISSUE TITLE
translate to a point?

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cpptraj#344]
what do you think about adding translate COG or COM of frame with given mask to a point?

I used to use Chimera to rotate/translate one of the molecules to a specific position and then save to new pdb file with others.

pytraj syntax
```python
traj.translate('@1', to=[0, 10, 1.])
traj2.traj.translate('@1', to=[0, 10, 1.])

# combine two trajectories and visualize it.
# can use `rotate` too.
(traj + traj2).view()
```

**Comment by hainm** [comment]
current cpptraj usage
```
        [<mask>] [x <dx>] [y <dy>] [z <dz>]
  Translate atoms in <mask>
```

**Comment by drroe** [comment]
I think such functionality would be best implemented in the 'center'
command.

On Friday, May 27, 2016, Hai Nguyen <notifications@github.com> wrote:

> current cpptraj usage
>
>         [<mask>] [x <dx>] [y <dy>] [z <dz>]
>   Translate atoms in <mask>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Amber-MD/cpptraj/issues/344#issuecomment-222291290>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJrJt4uoyCbLLn-a2Oc7o6iYoWNqfS4oks5qF9MEgaJpZM4IpADP>
> .
>


-- 
-------------------------
Daniel R. Roe, PhD
Department of Medicinal Chemistry
University of Utah
30 South 2000 East, Room 307
Salt Lake City, UT 84112-5820
http://home.chpc.utah.edu/~cheatham/
(801) 587-9652
(801) 585-6208 (Fax)

**Comment by hainm** [comment]
Yeah right.

**Comment by drroe** [comment]
Addressed by #348

**Comment by hainm** [comment]
thanks. I tried dummy test and seems work well.

![screenshot-3](https://cloud.githubusercontent.com/assets/4451957/15946550/52bbe7ca-2e65-11e6-84ad-73e94a6198b9.png)
